src/test/java/org/broadinstitute/hellbender/engine/MultiVariantWalkerGroupedOnStartUnitTest.java

```diff
@@ -133,7 +133,7 @@ public void testMergingTwoFilesCorrectBehavior() {
             List<String> inputList1 = ((List<VariantContext>)IteratorUtils.toList(variantContextFeatureDataSource1.iterator())).stream().map(VariantContext::toString).collect(Collectors.toList());
             List<String> inputList2 = ((List<VariantContext>)IteratorUtils.toList(variantContextFeatureDataSource2.iterator())).stream().map(VariantContext::toString).collect(Collectors.toList());
 
-            Assert.assertEquals(tool.seenVariants.size(), 14);
+            Assert.assertEquals(tool.seenVariants.size(), 13);
```

Comment:
So the previous results without -R were just silently incorrect?

Reply:
No, I changed the test file which had a read on chromosome 19